pkg/virtctl/console/console.go

```diff
@@ -209,82 +107,63 @@ func TerminalWebsocketCallback(ws *websocket.Conn, resp *http.Response, connErr
 		close(stopChan)
 	}()
 
-	// If there is no obvious connection error, set up the terminal
-	if connErr == nil {
-		state, err := terminal.MakeRaw(int(os.Stdin.Fd()))
-		if err != nil {
-			return fmt.Errorf("Make raw terminal failed: %s", err)
+	go func() {
+		defer close(readStop)
+		buf := make([]byte, 1024, 1024)
```

Comment:
It should be possible to use `io.Copy` instead of the manual copying between the reader and the writer.

Reply:
yup, it is possible. I've updated this logic to use io.copy when possible.